testing/setup-googletest.h

```diff
@@ -36,5 +36,7 @@
     /* Turn off virtual_io's input. */ \
     Kaleidoscope.device().keyScanner().setEnableReadMatrix(false); \
     testing::InitGoogleTest(); \
-    RUN_ALL_TESTS(); \
+    /* We assign the return value of RUN_ALL_TESTS, to */ \
+    /* silence a compiler warning. */                     \
+    int __ignore__ = RUN_ALL_TESTS();           \
```

Comment:
We can't use `NOLINT` here, because this is a warning issued by gcc, not cpplint.